packages/webpack.config.js

```diff
@@ -138,11 +137,6 @@ function getJavaScriptWebpackConfig() {
         },
       ],
     },
-    plugins: [
-      new webpack.optimize.UglifyJsPlugin({
```

Comment:
why did you remove this?

Reply:
This is breaking changes in webpack v4.
We can configure related configuration in `optimization`.
(https://webpack.js.org/configuration/optimization/)
In production mode, webpack4 minize bundle file by default.
So I remove this line :)

Reply:
I'm going to test this, but I don't want to break anyone by changing the the file name

Reply:
It looks like both the .min.js and .js files are the same, which means it doesn't break anyone. I think we should eventually deprecate the .min.js files.

Reply:
You're right.
I'll configure to not generate min file!

Reply:
Cool - this will be a breaking change for people...Can you make sure to add that in the PR description.

Reply:
I reflect it!
If necessary, we can also remove css.min files.